SPECS/cert-manager/cert-manager.spec

```diff
@@ -3,10 +3,29 @@ Version:        1.5.3
 Release:        1
 Summary:        Automatically provision and manage TLS certificates in Kubernetes 
 License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
 URL:            https://github.com/jetstack/cert-manager
 #Source0:       https://github.com/jetstack/%{name}/archive/refs/tags/v%{version}.tar.gz
 Source0:        %{name}-%{version}.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated external dependencies from this tarball, since network is disabled during build time.
+#   1. wget https://github.com/jetstack/%{name}/archive/refs/tags/v%{version}.tar.gz -o %%{name}-%%{version}.tar.gz
+#   2. tar -xf %%{name}-%%{version}.tar.gz
+#   3. cd %%{name}-%%{version}
+#   4. patch -p1 < Fix-dependency-checksum.patch
+#   5. mkdir -p BAZEL_CACHE
+#   6. bazel fetch --repository_cache=BAZEL_CACHE //...
+#   7. tar -czvf %{name}-%{version}-vendor.tar.gz BAZEL_CACHE
```

Comment:
Simply compressing it this way will leave a bunch of metadata, which will be different every time you compress the same folder again. That will cause us to generate a different hash each time and never have the confidence, that we've got the same sources. Take a look at the options we've used in `coredns` for instance - they make sure you always end up with the same tarball.

Reply:
Fixed.